sapi/fpm/fpm/fpm_systemd.c

```diff
@@ -29,13 +29,13 @@ static void fpm_systemd(void)
 	}
 
 /*
-	zlog(ZLOG_DEBUG, "systemd %s (Processes active:%d, idle:%d, Requests:%lu, slow:%lu, Traffic:%.3greq/sec)",
+	zlog(ZLOG_DEBUG, "systemd %s (Processes active:%d, idle:%d, Requests:%lu, slow:%lu, Traffic:%ureq/sec)",
 			fpm_global_config.systemd_watchdog ? "watchdog" : "heartbeat",
 			active, idle, requests, slow_req, ((float)requests - last) * 1000.0 / fpm_global_config.systemd_interval);
 */
 
 	if (0 > sd_notifyf(0, "READY=1\n%s"
-				"STATUS=Processes active: %d, idle: %d, Requests: %lu, slow: %lu, Traffic: %.3greq/sec",
+				"STATUS=Processes active: %d, idle: %d, Requests: %lu, slow: %lu, Traffic: %ureq/sec",
```

Comment:
`%u` is not correct: The value is a float and it's probably useful to show the decimal digits for small numbers. I would suggest `%.3f`: This always shows exactly 3 decimal digits [1] and will not use the scientific notation.

[1] It will show trailing zeros, but those are pretty unlikely anyways.

Reply:
I somehow glossed over use-cases with < 1req/s.
Always using 2 decimal digits make sense.